### DISCUSSION TITLE
What about the "MIssing Menubar" issue?

### DISCUSSION THREAD

**Discussion by antoniograzioli** [RomanKubiak/ctrlr#365] [General]
I have just downloaded the newer version (5.6.23) and discovered that the "menubar" is always hidden on "Restricted Istances" - at least on my Mac
That makes the Standalone or the plugin version useless, of course.
Is there any news on this issue? A new release coming up? 
Any hint on how to make it work or a "last known good" release ?

**Comment by Godlike-Productions** [comment]
If you just need the MIDI menu, you can call this from a uiButton by setting componentInternalFunction to MIDI Device Selection.

**Reply by antoniograzioli** [reply]
cool, many thanks! I didn't know about this "MIDI Device Selection" function!  That's exacly what I was looking for! Thanks again

**Comment by bijlevel** [comment]
I'm aware of the missing menu bar (and notified Roman Kubiak in the Issue section), but this can be also a blessing in disguise. If you add the midi menu AND you provide 'zoom in/out' then that's probably all you need, a very simple UI. If the standalone (and this goes for all other plugins) came with a (restricted) menu bar, it still has rather a lot of options and probably confuses the user.

**Comment by Godlike-Productions** [comment]
There is a LUA command to enable and disable the menu in the later versions as well.  I can't remember what it is, but I tested it a while back.  It may be in one of the demo panels

**Comment by antoniograzioli** [comment]
Thanks for all the precious infos! in the meanwhile I simply added a button for "MIDI Settings" and this is exactly what I was missing. Thx again!

![immagine](https://user-images.githubusercontent.com/16060683/136650212-051965e3-aac2-4301-bab6-e61817f54bbc.png)
